packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/utils/included_fields.ts

```diff
@@ -39,14 +35,13 @@ export function includedFields(
     return;
   }
 
-  // convert to an array
-  const sourceFields = toArray(fields);
-  const sourceType = toArray(type);
+  const sourceFields = typeof fields === 'string' ? [fields] : fields;
+  const sourceType = typeof type === 'string' ? [type] : type;
 
   return sourceType
     .reduce((acc: string[], t) => {
-      return [...acc, ...sourceFields.map((f) => `${t}.${f}`)];
+      acc.push(...sourceFields.map((f) => `${t}.${f}`));
+      return acc;
     }, [])
-    .concat(ROOT_FIELDS)
-    .concat(fields); // v5 compatibility
```

Comment:
So this is the only behavioral change from the PR: I removed the 'v5' compatibility from the time the type fields were at the top level. (this should have been removed years ago...)